datadog/metric_test.go

```diff
@@ -33,23 +33,17 @@ import (
 	"github.com/golang/mock/gomock"
 	"github.com/golang/protobuf/proto"
 	ddapi "github.com/zorkian/go-datadog-api"
-	"google.golang.org/appengine/aetest"
 	metricpb "google.golang.org/genproto/googleapis/api/metric"
 	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
 )
 
 var testCtx context.Context
 
 func TestMain(m *testing.M) {
-	var done func()
-	var err error
-	testCtx, done, err = aetest.NewContext()
-	if err != nil {
-		panic(err)
-	}
-
+	DSStop, _ := datastore.Emulator(8081)
+	testCtx = context.Background()
```

Comment:
I don't think you need a shared test context anymore, so perhaps each test should just call `context.Background()` independently.

Reply:
Fair point, rewrote to use separate contexts.